content/applications/marketing/surveys/dashboard.rst

```diff
@@ -0,0 +1,178 @@
+=================
+Surveys dashboard
+=================
+
+Companies gather valuable insights from customers and employees through surveys, aiding informed
+decision-making.
+
+With Odoo *Surveys*, users create various surveys, questionnaires, certifications, assessments, and
+so much more. These can be used to collect feedback, evaluate the success of a recent event, and
+measure the satisfaction of customers and/or employees. This process yields valuable insights into
+shifting market trends.
+
+Dashboard
+=========
+
+Upon opening the *Surveys* application, Odoo presents the main dashboard of the *Surveys*
+application, otherwise known as the :guilabel:`Surveys` page.
+
+.. image:: dashboard/surveys-dashboard.png
+   :align: center
+   :alt: The Odoo Surveys application dashboard in the default kanban view.
+
+.. tip::
+   The *Surveys* dashboard can be accessed at any time throughout the application by clicking
+   :menuselection:`Surveys` from the header menu.
+
+In the upper-left corner, there is a :guilabel:`New` button. When clicked, Odoo presents a blank
+survey form that can be used to :doc:`create a survey <../surveys/create>`.
+
+On the dashboard, all the surveys that have been created in the database are displayed in a default
+Kanban view.
+
+From left-to-right, after the survey name, the user who is responsible for it, and the month it was
+created, each line on the *Surveys* dashboard shows the following:
+
+- Number of :guilabel:`Questions` in that particular survey
+- :guilabel:`Average Duration` of the survey (how long it typically takes a participant to complete)
+- Number of :guilabel:`Registered` participants for the survey
+- Number of times that particular survey has been :guilabel:`Completed`
+- Percentage and data bar showcasing how many people have :guilabel:`Passed` or become
+  :guilabel:`Certified`
+
+.. note::
+   A half-trophy background image behind the survey name indicates that the survey is a
+   *Certification*.
```

Comment:
Suggesting to move this note below the bullet points since this is a minor visual cue that is added to Certification surveys, whereas the "Certified" percentage is a stronger visual cue that the record is a Certification survey
```suggestion
```